src/main/java/invtweaks/InvTweaksConfigManager.java

```diff
@@ -215,8 +217,9 @@ private boolean loadConfig() {
         }
     }
 
-    private boolean extractFile(ResourceLocation resource, File destination) {
-        try(InputStream input = mc.getResourceManager().getResource(resource).getInputStream()) {
+    private boolean extractFile(String resourcePath, File destination) {
+        ClassLoader classloader = Thread.currentThread().getContextClassLoader();
+        try (InputStream input = classloader.getResourceAsStream(resourcePath)) {
```

Comment:
> A quick test is to see if the translations load, try opening the inventory tweaks configuration screen. If the text is all correct and not a bunch of 'invtweaks.*.*' type stuff but the resource-system config loader doesn't work, then something probably changed in 1.11 that I'd have to look up to figure out. If the text is all messed up, then the resource system isn't being set up correctly so it's most likely breaking because of that.

I will test it this evening, if it works without my resource loader changes and a packaged jar and will report here.
But if I get the `invtweaks.*` keys on the settings, maybe the changed resource loading is acceptable for now.